content/en/docs/installation/uninstall.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Uninstall the notation CLI and its configuration
+description: Uninstall the notation CLI, plugins, and configuration files
+weight: 2
+---
+
+The `notation` CLI uses the following directories for its binaries and configuration files:
+
+- `NOTATION_BIN` is the directory for the `notation` binary.
```

Comment:
Are those env variables? They look like such... Shouldn't by convention they use $ in documentation? Are they automatically set during the installation? We should clarify that.

Reply:
Based on my testing, you can set these as environment variables to update the default location, but they are not set by default. `notation` seems to use a default value per platform if no value is present.

Updated the IMPORTANT callout to reflect this.